src/Transport/Addressing/Partitioning/Strategies/FailOverNamespacePartitioning.cs

```diff
@@ -11,10 +11,17 @@ public class FailOverNamespacePartitioning : INamespacePartitioningStrategy
 
         public FailOverNamespacePartitioning(ReadOnlySettings settings)
         {
-            if (!settings.TryGet(WellKnownConfigurationKeys.Topology.Addressing.Partitioning.Namespaces, out namespaces) || namespaces.Count != 2)
+            if (!settings.TryGet(WellKnownConfigurationKeys.Topology.Addressing.Partitioning.Namespaces, out namespaces))
             {
                 throw new ConfigurationErrorsException("The 'FailOver' namespace partitioning strategy requires exactly two namespaces to be configured");
```

Comment:
The approach we took is to use name of the strategy as-is. Should we do the same?
```c#
throw new ConfigurationErrorsException($"The '{nameof(FailOverNamespacePartitioning)}' namespace partitioning strategy requires exactly two namespaces to be configured");
```